README.rst

```diff
@@ -60,7 +60,7 @@ latest version:
 
 .. code:: bash
 
-    $ pip install cartoframes
+    $ pip install (--pre) cartoframes
```

Comment:
This should be `pip install --pre cartoframes` or `pip install cartoframes==1.0b1`

Reply:
If you want to add the two options, I would add it explicitly A or B

Reply:
Yeah, as someone who suffers from not understanding code documentation, I support being very explicit